src/languages/ko-KR.js

```diff
@@ -230,8 +230,8 @@ export default {
     "Channel Name": "채널 이름",
     "Uptime Kuma URL": "Uptime Kuma URL",
     aboutWebhooks: "Webhook에 대한 설명: {0}",
-    aboutChannelName: "Webhook 채널을 우회하려면 {0} 채널 이름칸에 채널 이름을 입력해주세요. 예: #기타-채널",
-    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Project Github 페이지로 설정해요.",
+    aboutChannelName: "Webhook 채널을 무시하려면 {0} 채널 이름칸에 채널 이름을 입력해주세요. 예: #기타-채널",
+    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Github Project  페이지로 설정해요.",
```

Comment:
```suggestion
    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Github Project 페이지로 설정해요.",
```

Edit multiple space.